agent/taskresource/cgroup/control/cgroupv2_controller_linux.go

```diff
@@ -0,0 +1,79 @@
+//go:build linux
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package control
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/cihub/seelog"
+	cgroupsv2 "github.com/containerd/cgroups/v2"
+)
+
+const (
+	defaultCgroupv2Path = "/sys/fs/cgroup"
+	parentCgroupSlice   = "/"
+)
+
+// controlv2 is used to implement the cgroup Control interface
+type controlv2 struct{}
+
+// Create creates a new cgroup based off the spec post validation
+func (c *controlv2) Create(cgroupSpec *Spec) error {
+	// Validate incoming spec
+	err := validateCgroupSpec(cgroupSpec)
+	if err != nil {
+		return fmt.Errorf("cgroup create: failed to validate spec: %w", err)
+	}
+
+	cgroupPath := cgroupSpec.Root
+	seelog.Infof("Creating cgroup cgroupv2root=%s parentSlice=%s cgroupPath=%s", defaultCgroupv2Path, parentCgroupSlice, cgroupPath)
+	_, err = cgroupsv2.NewSystemd(parentCgroupSlice, cgroupPath, -1, cgroupsv2.ToResources(cgroupSpec.Specs))
```

Comment:
nit: for readability, can we put that `-1` in some constant that describes that it's for?